config/helm/chart/default/tests/Common/kubernetes-monitoring/clusterrole-kubernetes-monitoring_test.yaml

```diff
@@ -90,3 +90,95 @@ tests:
               - /livez
             verbs:
               - get
+
+  - it: should add additional permissions when flag is enabled
+    set:
+      additionalPermissions.pvcMonitoring: true
+    asserts:
+      - isKind:
+          of: ClusterRole
+      - equal:
+          path: metadata.name
+          value: dynatrace-kubernetes-monitoring
+      - isNotEmpty:
+          path: metadata.labels
+      - isNotEmpty:
+          path: rules
+      - contains:
+          path: rules
+          content:
+            apiGroups:
+              - ""
+            resources:
+              - nodes
+              - pods
+              - namespaces
+              - replicationcontrollers
+              - events
+              - resourcequotas
+              - pods/proxy
+              - nodes/proxy
+              - services
+              - nodes/metrics
+            verbs:
+              - list
+              - watch
+              - get
```

Comment:
only this assert is needed as the other ones are already checked in previous tests